doc/api/module.md

```diff
@@ -175,6 +175,28 @@ globalPreload: http-to-https
 globalPreload: unpkg
 ```
 
+This function can also be used to pass data to the loader's [`initialize`][]
+hook include transferrable objects like ports.
```

Comment:
This sentence seems weird to me (non-native English speaker)

Reply:
Oops `include` should be `including` 😅